app/src/main/java/com/doctor/yumyum/App.kt

```diff
@@ -0,0 +1,18 @@
+package com.doctor.yumyum
+
+import android.app.Application
+import android.content.SharedPreferences
+
+class App : Application() {
+    companion object {
+        lateinit var sharedPref: SharedPreferences
+    }
```

Comment:
```suggestion
 lateinit var sharedPref: SharedPreferences
```
preference를 상수화할 필요는 없어보입니다. 제거를 권유드려요!